rct229/ruletest_engine/ruletest_jsons/ashrae9012019/section11/rule_11_6.json

```diff
@@ -0,0 +1,131 @@
+{
+    "rule-11-6-a": {
+        "Section": 11,
+        "Rule": 6,
+        "Test": "a",
+        "test_description": "A one story building, single zone building has service hot water piping. The service hot water piping losses are correctly modeled.",
+        "expected_rule_outcome": "pass",
+        "standard": {
+            "rule_id": "11-6",
+            "ruleset_reference": "Table G3.1 #11, baseline column, i",
+            "rule_description": "Piping losses shall not be modeled.",
+            "applicable_rmr": "Baseline Model",
+            "rule_assertion": "=",
+            "comparison_value": "Expected Value",
+            "mandatory_rule": "No",
+            "schema_version": "0.0.34"
+        },
+        "rmd_transformations": {
+            "baseline": {
+                "id": "ASHRAE229 1",
+                "calendar": {
+                    "is_leap_year": false
+                },
+                "data_timestamp": "2024-02-12T12:00Z",
+                "ruleset_model_descriptions": [
+                    {
+                        "id": "RMD 1",
+                        "buildings": [
+                            {
+                                "id": "Building 1",
+                                "building_segments": [
+                                    {
+                                        "id": "Building Segment 1",
+                                        "zones": [
+                                            {
+                                                "id": "Thermal Zone 1",
+                                                "floor_name": "Floor1",
+                                                "spaces": [
+                                                    {
+                                                        "id": "Space 1",
```

Comment:
I believe we need the ```service_water_heating_uses``` key here (please see Jiarong's PR- https://github.com/pnnl/ruleset-checking-tool/pull/1511/files#diff-9214518a5c95ad02287645995653e70e7c6ee9b0f68a9bd81bfb8d27c68d47bb:~:text=%3A%20%22Space%201%22%2C-,%22service_water_heating_uses%22%3A%20%5B,-%7B%22id%22%3A%20%22service).

Reply:
Great catch, thanks Yun Joon. The service water heating distribution should know be linked in the ruletest